config.sample.py

```diff
@@ -1,5 +1,20 @@
 import logging
 
+stats = {
+    # stats settings
+    # settings in this section will take effect on the fly. no need to restart
+    # ------------------------------------------------------------------------
+
+    # how long to sleep in seconds between stats reports
+    'sleep_time': 300,
+
+    # print header every nth report
+    'header_every_nth': 20,
+
+    # write a separate .csv file for use in excel.
+    'write_csv': True,
```

Comment:
I realized I should put stats.py in scripts and update the README.  I'll do that now.